examples/petsc/bpsswarm.c

```diff
@@ -302,14 +303,9 @@ int main(int argc, char **argv) {
     PetscCall(KSPGetPC(ksp, &pc));
     if (bp_choice == CEED_BP1 || bp_choice == CEED_BP2) {
       PetscCall(PCSetType(pc, PCJACOBI));
-      PetscCall(PCJacobiSetType(pc, PC_JACOBI_ROWSUM));
+      PetscCall(PCJacobiSetType(pc, PC_JACOBI_DIAGONAL));
     } else {
       PetscCall(PCSetType(pc, PCNONE));
-      MatNullSpace nullspace;
-
-      PetscCall(MatNullSpaceCreate(PETSC_COMM_WORLD, PETSC_TRUE, 0, 0, &nullspace));
-      PetscCall(MatSetNullSpace(mat_O, nullspace));
-      PetscCall(MatNullSpaceDestroy(&nullspace));
```

Comment:
I made the two match. Alternatively, we could add this to `bps.c`?

Reply:
Ah, I think I copied from `bpssphere.c` originally -- if this isn't needed here I'm good with removing.